app/styles/modules/_settings.scss

```diff
@@ -809,3 +809,113 @@ section.modal-panel {
     }
   }
 }
+
+.security-panel {
```

Comment:
hm suggestion: can we move all the security panel into its own file to something like `_settings_security.scss` ?